sshnode.go

```diff
@@ -48,17 +49,17 @@ func NewVagrantNode(name, port, privKeyFile string) (*VagrantNode, error) {
 	}
 
 	config := &ssh.ClientConfig{
-		User: "vagrant",
+		User: user,
```

Comment:
I had thought about it but I intentionally chose not to store it as it is used only once while preparing the config, so it's scope is very local to this function at this point. Did you want to store username info for some other purpose?